libbeat/outputs/kafka/config.go

```diff
@@ -209,6 +211,12 @@ func newSaramaConfig(config *kafkaConfig) (*sarama.Config, error) {
 	// configure per broker go channel buffering
 	k.ChannelBufferSize = config.ChanBufferSize
 
+	// configure bulk size
+	if config.BulkFlushFrequency > 0 {
+		k.Producer.Flush.Messages = config.BulkMaxSize
```

Comment:
Should we set `MaxMessages` instead so this is consequent with current description and the expected implementation of other inputs?
```suggestion
		k.Producer.Flush.MaxMessages = config.BulkMaxSize
```

Reply:
changed